userguide/tutorials/getting_started.adoc

```diff
@@ -136,47 +136,58 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
+
+1. The rest of the guide assumes that you have installed Tomcat in a directory called *CATALINA_HOME*.
 
-* Unlike other installation methods, you will also need to install and configure your database manually. See https://docs.killbill.io/latest/userguide_deployment.html#_database_engine[our deployment guide] for more details.
-* Installation on Windows might work, but isn't tested nor recommended by the core team.
-* We recommend installing the https://tomcat.apache.org/native-doc/[Apache Tomcat Native Library] (automatically done for you when using Ansible, Docker, or in our Cloud Marketplace instances).
 
 ==== Tomcat configuration
 
-Recommended Tomcat configuration files can be found https://github.com/killbill/killbill-cloud/tree/master/ansible/templates/tomcat/conf[here]. The `.j2` extension is for the Jinja templating language: you must render the files first using your custom variables, which depend on your environment (take a look at an example of such variables https://github.com/killbill/killbill-cloud/blob/master/ansible/group_vars/all.yml[here]). https://ansible.sivel.net/test/[This website] can be used to perform the rendering online if you do not have Jinja installed.
+1. Download the Recommended Tomcat configuration files from  https://github.com/killbill/killbill-cloud/tree/master/ansible/templates/tomcat/conf[here].The configuration files have the `.j2` extension for the Jinja templating language. 
+
+1. Render the files  using your custom variables (take a look at an example of such variables https://github.com/killbill/killbill-cloud/blob/master/ansible/group_vars/all.yml[here]). The , custom variables depend on your environment.  https://ansible.sivel.net/test/[This website] can be used to perform the rendering online if you do not have Jinja installed.
 
-Once rendered, `setenv2.sh` should be placed in *CATALINA_HOME/bin* and all other files should be placed in *CATALINA_HOME/conf*.
+1. Place the `setenv2.sh` in *CATALINA_HOME/bin* 
+
+1. Plase the other files in *CATALINA_HOME/conf*.
 
 Using these default configuration files, Tomcat will start on port 8080, which should be the only port open for ingress traffic in production (egress traffic should not be restricted).
 
-==== Kill Bill
+==== Kill Bill Configuration
 
-The Kill Bill configuration properties need to be added at the end of the file *CATALINA_HOME/conf/catalina.properties*. The minimal set of system properties required is the ones for the jdbc parameters (refer to our http://docs.killbill.io/latest/userguide_configuration.html[configuration guide] for more details). For example (update the values as needed):
+1. Add the Kill Bill configuration properties at the end of the *CATALINA_HOME/conf/catalina.properties* file.  At the  minimum, the  JDBC  parameters need to be added. So, for example, you can add the following(update the values as needed):
+
+ org.killbill.dao.url=jdbc:mysql://127.0.0.1:3306/killbill
```

Comment:
You might want to encapsulate this in a code block (`----`) as it's easier for the reader to copy / paste.

Reply:
asciidoc will again do the right thing here with the indent. The drawback with the code block is that it messes the numbering, so we will keep it that way.

Reply:
Not added the code block as it messes up the numbering